docs/guides/cloud/data-storage-and-masking.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: Data Storage and Controls
+sidebar_position: 60
+---
+
+:::info
+
+## <Icon name="graduation-cap" /> What you'll learn
+
+- What data Cypress Cloud stores
+- What controls exist to control data sent to and viewed in Cypress Cloud
```

Comment:
```suggestion
- What controls exist to manage data sent to and viewed in Cypress Cloud
```
minor: change up a word to avoid "what controls... control"

Reply:
Addressed in [`06a38a9` (#5425)](https://github.com/cypress-io/cypress-documentation/pull/5425/commits/06a38a963c8df47c79ad35fcafc4a29b616249a1)